test/e2e/data/infrastructure-docker/cluster-template/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
```

Comment:
from @sedefsavas https://github.com/kubernetes-sigs/cluster-api/pull/3639#discussion_r489150280

> nit. We can generate cluster-template.yaml on the fly and use that as a base to avoid extra changes we will need to do here when apis change.

I'm not sure to fully understand your suggestion...

Reply:
Similar to how CAPV is doing:

https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/de70ff05039df7ab15a4ba11a3f559f4e92f9df7/Makefile#L120

Generate cluster-temlate.yaml and then use that as base, then generate e2e specific templates using Kustomize.